### DISCUSSION TITLE
How to change favicons?

### DISCUSSION THREAD

**Discussion by muditporwal** [binokochumolvarghese/lightbi-hugo#47] [Q&A]
How can one change favicons ? Even the demo site has the default favicon.

**Comment by binokochumolvarghese** [comment]
you can go to the static folder and update the favicon as per your preferences. Check out the demo [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo/tree/main/themes/lightbi-hugo/static/favicon).

**Comment by muditporwal** [comment]
I created a directory ${project_root}/static/favicons and placed my icons there and it reflected. Hugo created the override based on the folder outside.

Thanks for pointing out.
